### PR TITLE
Make nice plots

### PR DESCRIPTION
**Pull request by matthiasto**
Make similarity solution presentation plots

### CODE REVIEW

File: avaframe/out3Plot/outAna1Plots.py [hunk 1]
```diff
@@ -337,67 +336,44 @@ def plotErrorConvergence(simDF, outDirTest, cfgSimi, xField, yFieldArray, colore
...
-    plt.show()
-    return fig1, ax1, ax2, slopeU, slopeH
+    # fig2, ax = plt.subplots(figsize=(2*pU.figW, 2*pU.figH))
+    # for sizeValue, simDFrow in slopeU.iterrows():
+    #     ax.scatter(slopeU.columns, simDFrow, label=sizeValue)
+    # plt.legend(title=sizedBy)
+    # ax.set_xlabel(coloredBy)
+    # ax.set_ylabel('Slope')
+    # ax.set_title('Speed of convergence of the DFA solution based on' + getTitleError(relativ, r' L2 on flow thickness'))
+    # plt.show()
```
Comment (fso42) [inline avaframe/out3Plot/outAna1Plots.py:375]: Is this still needed?

File: avaframe/out3Plot/outAna1Plots.py [hunk 2]
```diff
@@ -488,6 +464,109 @@ def plotErrorRef(simDF, outDirTest, cfgSimi, xField, yFieldArray, coloredBy, siz
...
+            hErrorL2 = simDFNewNew[yField]
+            if count >= 1:
+                simDFNewNew = simDFNewNew.append(simDFOld)
+            colorList = simDFNewNew[coloredBy].unique()
+            lenColor = len(colorList)
+            if lenColor<=10:
+                lenColor = None
+            # make the scatter plot
+            scatter1 = ax1.scatter(simDFNewNew[xField], simDFNewNew[yField], c=simDFNewNew[coloredBy], s=sizeList, cmap=cmap, norm=norm,
+                                   marker=pU.markers[0], alpha=1)#, edgecolors='k')
```
Comment (fso42) [inline avaframe/out3Plot/outAna1Plots.py:532]: Either use it or remove the edgecolor comment

**Review by fso42 — APPROVED** [review]
Minor changes, please fix before merging

File: avaframe/out3Plot/outAna1Plots.py [hunk 3]
```diff
@@ -488,6 +456,109 @@ def plotErrorRef(simDF, outDirTest, cfgSimi, xField, yFieldArray, coloredBy, siz
...
+        column of the simDF to use for the x axis
+    yField: str
+        column of the simDF to use for the y axis
+    coloredBy: str
+        column of the simDF to use for the colors
+    sizedBy: str
+        column of the simDF to use for the marker size
+    logScale: boolean
+        If you want a loglog scale
+    """
```
Comment (awirb) [inline avaframe/out3Plot/outAna1Plots.py:481]: add fit parameter

**Review by awirb — CHANGES_REQUESTED** [review]

**Pull request merged**